Samples~/AvatarCreatorSamples/AvatarCreatorElements/Scripts/SimpleAvatarCreator.cs

```diff
@@ -86,8 +72,58 @@ public async void SaveAvatar()
             loading.SetActive(false);
         }
 
+        public void CreateSecondTemplateAvatar()
+        {
+            CreateTemplateAvatar();
+        }
+
+        public void LoadAvatarFromTemplate(IAssetData template)
+        {
+            LoadAvatarFromTemplate(template.Id);
+        }
+
+        public async Task<AvatarProperties> LoadAvatarFromTemplate(string templateId)
+        {
+            loading.SetActive(true);
+            var templateAvatarProps = await avatarManager.CreateAvatarFromTemplate(templateId, bodyType);
+
+            // Destroy the old avatar and replace it with the new one.
+            if (avatar != null)
+            {
+                Destroy(avatar);
+            }
+            var previousGender = gender;
+            avatar = templateAvatarProps.Item1;
```

Comment:
Some IDE's show return types but not all.  
It looks clear for us because we know what they are. 
If you are trying to understand the code from a Git PR for example it is very difficult :D 
If it was something hidden from developers it wouldn't be a problem but our public API's return the tuples.

Reply:
Hmm, yeah that makes sense. I'll create a task for it.

Reply:
But it's breaking change, so we need a new version bump at some point for this.